include/effects_internal.h

```diff
@@ -59,7 +59,7 @@ FX_MAIN(lens_flare_main);
 FX_MAIN(got_item_outline_main);
 FX_MAIN(spiky_white_aura_main);
 FX_MAIN(smoke_impact_main);
-FX_MAIN(damage_indicator_main);
+void damage_indicator_main(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, f32 arg5, s32 arg6, EffectInstance **effectOut);
```

Comment:
I'd prefer to keep these args unnamed here for now since we don't know what they all are except for arg6, and also left align the ptrs please